package.json

```diff
@@ -138,14 +138,14 @@
         "upgrade:pdfjs": "./build/upgrade_pdfjs.sh && ./build/minify_pdfjs.sh"
     },
     "lint-staged": {
-        "tests/**/*.js": [
-            "prettier-eslint --print-width 120 --single-quote --tab-width 4 --write",
-            "git add"
-        ],
-        "src/lib/**/*.js": [
-            "prettier-eslint --print-width 120 --single-quote --tab-width 4 --write",
-            "git add"
-        ]
+        "linters": {
+            "*.js": ["eslint --fix", "git add"],
```

Comment:
Will this still apply the prettier rules, or are those same rules now in eslint?